backend/src/models/reports/Tab_50_rpt_PF_NetRecoverySummaryByQuarter.js

```diff
@@ -1,96 +1,100 @@
 const dbConnection = require("@database/databaseConnection");
 const { knex } = dbConnection();
-const _ = require("lodash");
+const { fiscalYearTable } = require("@models/useDbTables");
 
-// get the fiscal year based on the id passed from frontend
-const getFiscalYear = (fiscal) =>
-  knex.select(knex.raw(`fiscal_year from data.fiscal_year`)).where({ "fiscal_year.id": fiscal });
+// gets the current date in ISO "YYYY-MM-DD" format.
+const getCurrentDate = async () => new Date().toISOString().split("T")[0];
```

Comment:
I assume all reports use the same method for getting the current date. Could this be done in the generic controller then if all the reports will need it?